.vib/influxdb/runtime-parameters.yaml

```diff
@@ -1,32 +1,148 @@
-auth:
+image:
```

Comment:
Any reason for enabling debug? If so, add a comment as it is uncommon

Reply:
addressed at https://github.com/bitnami/charts/pull/33743/commits/859c509824da5be460e8f02a9701f49e440ae834